src/components/modal/v2/parts/views/LongTerm/Content.jsx

```diff
@@ -1,19 +1,26 @@
 /** @jsx h */
 import { h, Fragment } from 'preact';
 import { useState } from 'preact/hooks';
-import { useXProps, useServerData } from '../../../lib';
+import { useXProps, useServerData, useCalculator } from '../../../lib';
 import Calculator from '../../Calculator';
 import ProductListLink from '../../ProductListLink';
 import Instructions from '../../Instructions';
 import Button from '../../Button';
+import getComputedVariables from '../../../lib/getComputedVariables';
 
 export const LongTerm = ({
     content: { calculator, disclaimer, instructions, disclosure, linkToProductList, buttonText, cta },
```

Comment:
```suggestion
    content: { calculator, disclaimer, instructions, disclosure, linkToProductList, cta },
```
We can remove this now that it won't be used. If you scroll down you can see it's being passed into the Calculator component (around line 64) as ``` buttonText={buttonText}```. We can actually remove this as well.

Reply:
So when I deleted it from there, the Calculator component no longer knew what buttonText is.